CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [11.0.0]
+
+- Updated to NET 8 SDK container
```

Comment:
Also mention that you added support for running .NET 7 inside the build container

Reply:
Did the change